### PR TITLE
feat: Add option for output path

### PR DESCRIPTION
**Pull request by metaspace**
Allow output directory to be specified on command line. Bail if directory
exists. Create parent directories as necessary. Remove output directory global variable.

Signed-off-by: Andreas Hindborg <andreas.hindborg@wdc.com>

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -189,20 +180,36 @@ def main(argv):
     dev = ''
     container = 'docker'
     try:
-        opts, args = getopt.getopt(argv, "hd:c:r:p:b:l", ["dev=", "container=", "report", "plot=","benchmark=", "list_benchmarks"])
+        opts, args = getopt.getopt(
+            argv,
+            "hd:c:r:p:b:lo:",
```
Comment (MaisenbacherD) [inline run.py:185]: I think this line should be
```suggestion
            "hd:c:r:p:b:l:o",
```

**Review by MaisenbacherD — COMMENTED** [review]
Rest looks good to me :)

File: run.py (hunk 1 above)
Comment (metaspace) [reply run.py:185]: l does not take an argument ?

Comment (MaisenbacherD) [reply run.py:185]: Okay than this is resolved :)

**Pull request merged**